### PR TITLE
solution for jv-oop-advanced

### PR DESCRIPTION
**Pull request by vladis9**

### CODE REVIEW

File: src/main/java/core/basesyntax/Circle.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, radius: " + getArea()
+                + " sq.units, color: " + getColor());
+    }
+}
+
+
```
Comment (okuzan) [inline src/main/java/core/basesyntax/Circle.java:23]: extra empty lines

File: src/main/java/core/basesyntax/Figure.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
```
Comment (okuzan) [inline src/main/java/core/basesyntax/Figure.java:17]: line

File: src/main/java/core/basesyntax/FigureSupplier.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int RANDOM_VALUE = 15;
```
Comment (okuzan) [inline src/main/java/core/basesyntax/FigureSupplier.java:6]: - random value
- constant
something doesn't add up here :) Call it something like  limit, max-value

File: src/main/java/core/basesyntax/IsoscelesTrapezoid.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private static final int DIVIDER = 2;
```
Comment (okuzan) [inline src/main/java/core/basesyntax/IsoscelesTrapezoid.java:4]: not necessary, geometry rules won't change

**Review by okuzan — APPROVED** [review]
Really good job on your first try, you avoided all the usual mistakes!

File: src/main/java/core/basesyntax/IsoscelesTrapezoid.java (hunk 4 above)
Comment (vladis9) [reply src/main/java/core/basesyntax/IsoscelesTrapezoid.java:4]: i thought that it's magic number :)

**Pull request closed without merging**